apps/api/src/applicant/entity/ienapplicant.entity.ts

```diff
@@ -27,10 +27,9 @@ export class IENApplicant {
   @Column('varchar')
   name!: string;
 
-  // description: 'HMBC ATS system unique ID'
-  @Index({ unique: true })
-  @Column('bigint', { nullable: true, comment: 'HMBC ATS system unique ID' })
-  applicant_id?: number;
+  @Index({unique:true})
+  @Column('uuid',{nullable:true})
+  applicant_id?:string;
```

Comment:
Can't we use it as our `id`? I don't know why we need to create another id.

Reply:
That makes sense to me.  I can remove this, and update the migration.

Reply:
On second thought... removing the applicant_id column and replacing it with the id column will be a bit more than I expected.
<img width="266" alt="Screen Shot 2022-09-06 at 4 02 47 PM" src="https://user-images.githubusercontent.com/54554766/188755027-1c054c62-1ca3-4728-8cbe-dffcdd606dd9.png">